group_vars/docker-registry/storage_backend

```diff
@@ -1,17 +0,0 @@
----
-# default storage type is local (file).
-# storage_type: "local"
-
-# storage_type: "gcs"
-gcs_bucket: "gds-docker-registry"
-gcs_storage_path: "/"
-gcs_oauth2: false
-gcs_access_key: "{{ lookup('env', 'GCS_ACCESS_KEY') }}"
-gcs_secret_key: "{{ lookup('env', 'GCS_SECRET_KEY') }}"
```

Comment:
The gcs storage variables are now added to site-gcs.yml as suggested. This commit has been edited to include them.